spec.bs

```diff
@@ -412,22 +421,17 @@ To <dfn>append private state token issue request headers</dfn> given a [=/reques
  1. [=Associate the issuer=] |issuer| with |topLevel|.
  1. If the [=number of tokens=] for |issuer| is at least 500, return.
  1. Let |commitments| be the result of [=look up the key commitments|looking up the key commitments=] for |issuer|.
- 1. If |commitments| is null,  return.
- 1. Pass issuer public keys to cryptographic procedures. Reject if keys are
-         malformed.
-
-    Issue(187): How do we get the issuer public keys here?
-
- 1. [=Discard tokens=] from |issuer| that are signed with keys other than those from
+ 1. If |commitments| is null, return.
+ 1. Perform [=discard tokens=] with |issuer| that are signed with keys other than those from
```

Comment:
Watch out for merge conflicts with #207, make sure that you keep the right line for discard tokens (the one in #207)

Reply:
I think this got merged correctly now?